assign-object.ts

```diff
@@ -0,0 +1,11 @@
+import assign from './assign'
+import id from './identifier'
+import list from './list'
+import S from './sanitised'
+
+/** @returns assigment based on object key-values. */
+export const assignObject =
+  (kv: { [key: string]: unknown }): S =>
```

Comment:
No need to change anything, but something that might help you in the future.

Instead of having to specificy object with key and type of key `{ [key: string]: ...` you can use a utility type `Record` https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype

Reply:
So in this case `Record<unknown>`

Reply:
Yes, it could be `Record<string, unknown>`.